nemo/collections/nlp/modules/common/prompt_encoder.py

```diff
@@ -70,7 +70,7 @@ def __init__(
         self.prompt_embeddings.weight.requires_grad = False
 
         # Set fixed indicies for forward pass
-        self.register_buffer('indices', torch.LongTensor(list(range(self.total_virtual_tokens))))
+        self.register_buffer("indices", torch.LongTensor(list(range(self.total_virtual_tokens))), persistent=False)
```

Comment:
Probably a breaking change since older peft modules will have this but newer ones wont. Need to check

Reply:
good catch! will check!

Reply:
ok this will indeed break older p-tuning checkpoints. However, it is a nice "feature" because older checkpoints will need to be converted anyway to a new param naming format. In that conversion step (which need to be written) I will remove the indices.